pygmt/tests/test_makecpt.py

```diff
@@ -154,29 +126,6 @@ def test_makecpt_reverse_color_only(grid):
     return fig
 
 
-@pytest.mark.mpl_image_compare
-def test_makecpt_reverse_zsign_only(grid):
-    """
-    Use static color palette table with its z-value sign reversed.
-    """
-    fig = Figure()
-    makecpt(cmap="earth", reverse="z")
-    fig.grdimage(grid, projection="W0/6i")
-    return fig
-
-
-@pytest.mark.mpl_image_compare
-def test_makecpt_reverse_color_and_zsign(grid):
-    """
-    Use static color palette table with both its colors and z-value sign
-    reversed.
-    """
-    fig = Figure()
-    makecpt(cmap="earth", reverse="cz")
-    fig.grdimage(grid, projection="W0/6i")
-    return fig
```

Comment:
What do you think about keeping this test_makecpt_reverse_color_and_zsign since it's more advanced, and maybe removing test_makecpt_reverse_color_only instead?

Reply:
That's fine. I figured we wanted to keep `test_makecpt_reverse_color_only` since it was using a Boolean, so it seemed like a more Python-specific thing to test, whereas passing strings is testing passing a literal string to the GMT API.

Reply:
Hmm, you're right on the Boolean thing. How about we keep both then? I feel like `makecpt` is one of those GMT modules (like `basemap` and `plot`) we want to overtest rather than undertest. I get nervous about removing tests (goes against all my job training on code testing).

Reply:
Re-added the function in 571c1ef